torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -889,9 +889,10 @@ class FullyShardedDataParallel(nn.Module):
 
         backward_prefetch (Optional[BackwardPrefetch]):
             This is an experimental feature that is subject to change in the
-            the near future. It allows users to enable two different backward_prefetch
+            the near future. It allows users to enable two different backward prefetch
             algorithms to help backward communication and computation overlapping.
             Pros and cons of each algorithm is explained in the class ``BackwardPrefetch``.
+            (Default: ``BackwardPrefetch.BACKWARD_PRE``)
```

Comment:
nit: since we are defaulting to this, we should have a brief description of what it does and the implications.